projects/admin.py

```diff
@@ -27,15 +29,26 @@ class PublicationAdmin(ProjectFields, admin.ModelAdmin):
         "month",
         "author",
         "bibtex_source",
+        "doi",
         "link",
         "added_by_username",
         "entry_created_date",
+        "source",
+        "status",
+        "approved_with",
+        "scopus_citations",
+        "semantic_scholar_citations",
     )
 
     ordering = ["project__charge_code", "-year", "-entry_created_date"]
     list_display = ("title", "project_charge_code", "year", "entry_created_date")
 
 
+class ChameleonPublicationAdmin(admin.ModelAdmin):
```

Comment:
What exactly is this being used for? I see there is also `PublicationAdmin` and I don't really understand what `ChameleonPublicationAdmin` would be used for.

Reply:
Ah I see these are the ones that Chameleon has published.

Reply:
Were you able to check out the admin views? Docker was not cooperating with me and I was not able to look at them locally.